Update using-svgs.md to include a reference to React Native Babel Tra… (#16168)
expo/expo: docs/pages/ui-programming/using-svgs.md

Once we have a vector created inside a design program, like Figma, Illustrator, or Sketch, find the "export" menu and specify "SVG" as the export type. This will create an SVG file we can view in a code editor. Alternatively, these programs often allow right clicking on an element, then copying it as an SVG.

## Converting SVG files using a Babel transformer

[React Native SVG transformer](https://github.com/kristerkari/react-native-svg-transformer) allows compile-time transformation to make SVG files compatible with React.

Follow the installation steps to configure your Expo project to use this workflow. After your project is properly configured, you'll be able to use your local SVG files like this:

```javascript
import Logo from "./assets/logo.svg";

<Logo width={120} height={40} />
```

## Converting individual SVG files for React Native

> If you don't want to change your project configuration, you can alternatively use a tool such as [React-SVGR](https://react-svgr.com/playground/?native=true) to convert your SVG files. It takes an SVG as input then can transform it into another format, including a format that works with React.

Reviewer: ```suggestion
Alternatively, [React-SVGR](https://react-svgr.com/playground/?native=true) is a great tool to convert individual SVG files. It takes an SVG as input then can transform it into another format, including a format that works with React.
```
